sentry-android-integration-tests/sentry-uitest-android/src/androidTest/java/io/sentry/uitest/android/EnvelopeTests.kt

```diff
@@ -84,10 +84,14 @@ class EnvelopeTests : BaseUiTest() {
         relayIdlingResource.increment()
         relayIdlingResource.increment()
         val transaction = Sentry.startTransaction("e2etests", "test1")
+        Thread.sleep(5)
```

Comment:
Do we rely on this pause bring 5 seconds? If it were 1, or 500ms, would it still validate what it needs?
I worry of things getting flaky so checking on the need of exact value, but also on making the test slower than it needs to be

Reply:
it's 5 milliseconds, and it's added to avoid that multiple transactions have the same cpu timestamp, as they are collected with milliseconds precision

Reply:
I'll add a comment to make it clear

Reply:
Ok, i removed the sleep altogether :D